src/index.html

```diff
@@ -11,7 +11,81 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get"></form>
```

Comment:
```suggestion
    <script type="text/javascript" src="./main.js"></script>
    
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get"></form>
```
blank line between sibling tags 😉